tests/repository/test_github_repository.py

```diff
@@ -6,14 +6,15 @@
 from thipster.repository.exceptions import ModelNotFoundError
 
 repository = 'THipster/models'
+branch = 'remove-folder-name'
```

Comment:
Need to change it to main after merging ?

Reply:
After merging the THipster/models one yup